main.go

```diff
@@ -62,6 +65,9 @@ func main() {
 
 	go postLicense()
 	go updateLicense()
+	if !*telemetryDisabled {
+		go sendTelemetry()
+	}
```

Comment:
```suggestion
	go sendTelemetry(*telemetryDisabled)
```

I would move this check into `sendTelemetry` method's itself and turn it into a guard clause instead.

Reply:
done, good suggestion